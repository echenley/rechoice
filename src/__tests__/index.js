import * as modules from '../index'

describe('exports', () => {
  it('exports the correct modules', () => {
    expect(modules).toMatchSnapshot()
  })
})
