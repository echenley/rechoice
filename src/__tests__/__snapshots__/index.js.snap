// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports exports the correct modules 1`] = `
Object {
  "AsyncMultiSelect": [Function],
  "AsyncSelect": [Function],
  "MultiSelect": [Function],
  "Select": [Function],
}
`;
